### PR TITLE
SSCS-11057 Jitpack Snapshot & Commit Build

### PR DESCRIPTION
**Pull request by TeaWithLucas**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dattipoeHMCTS — APPROVED** [review]

File: .github/workflows/jitpack_snapshot_commit.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: JitPack Snapshot & Commit Build
+
+on:
+  pull_request:
+    branches:
+      - master
+      -
```
Comment (ss-evoco) [inline .github/workflows/jitpack_snapshot_commit.yml:7]: Is this line correct ? Looks wrong to me.

File: .github/workflows/jitpack_snapshot_commit.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          echo 'version tag: "${{env.target_version}}"'
+          
+          jitpack_url="https://jitpack.io/com/github/${{env.target_repo}}/${{env.target_version}}/"
+          echo 'Jitpack URL: ${jitpack_url}'
+          
+          # timeout in 30 seconds to avoid waiting for build
+          curl -s -m 30 ${jitpack_url} || true
+          echo 'See progress https://jitpack.io/#${{env.target_repo}}'
+        env:
+          target_branch: '${{steps.get_versions.outputs.branch}}'
```
Comment (ss-evoco) [inline .github/workflows/jitpack_snapshot_commit.yml:32]: Will this mean we will have to change our branch naming strategy ? Does it still have to be under 30 (?) characters to allow the preview pipeline to work ?

**Review by ss-evoco — CHANGES_REQUESTED** [review]

File: .github/workflows/jitpack_snapshot_commit.yml (hunk 1 above)
Comment (TeaWithLucas) [reply .github/workflows/jitpack_snapshot_commit.yml:7]: Shouldn't Matter, but fixed

File: .github/workflows/jitpack_snapshot_commit.yml (hunk 2 above)
Comment (TeaWithLucas) [reply .github/workflows/jitpack_snapshot_commit.yml:32]: No, that's CCD defs that has that limit

**Review by ss-evoco — APPROVED** [review]

**Pull request merged**